Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SolutionTemplate.sln.template

```diff
@@ -2,10 +2,16 @@
 # Visual Studio 15
 VisualStudioVersion = 15.0.28307.539
 MinimumVisualStudioVersion = 10.0.40219.1
-#PROJECT_TEMPLATE Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "<PROJECT_NAME>", "<PROJECT_RELATIVE_PATH>", "{<PROJECT_GUID>}"
+#PROJECT_TEMPLATE_START
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "<PROJECT_NAME_TOKEN>", "<PROJECT_RELATIVE_PATH_TOKEN>", "{<PROJECT_GUID_TOKEN>}"
```

Comment:
Does this GUID need to change per-solution (or does it somewhere?)?

Reply:
This is a Type Guid, the type here is C# Project file.